### PR TITLE
styler + tabs to spaces + typos

### PR DESCRIPTION
**Pull request by vnijs**
Applied [styler](http://styler.r-lib.org/) to all R code, converted tabs to 2 spaces where not yet applied, and corrected a few typos

### CODE REVIEW

**Comment by vnijs** [comment]
I also ran `devtools::revdep_check` and no issues there

**Comment by vnijs** [comment]
Fix for #59. See example below. Effects `fileGetter` in `filechoose.R`, DESCRIPTION (import from `methods`), and NAMESPACE

```r
library(shiny)
library(shinyFiles)

# options(shiny.trace = TRUE)

shinyApp(
  ui = fluidPage(
    sidebarLayout(
      sidebarPanel(
        uiOutput("shinyFilesOutput"),
        selectInput("filetype", "Choose file type", c("csv", "txt", "rda", "R")),
        textInput(
          "filepattern", "File pattern", value = "", 
          placeholder = "provide a regular expression to filter files"
        ),
        textInput(
          "filename", "Initial filename", value = "", 
          placeholder = "provide an initial filename"
        )
      ),
      mainPanel()
    )
  ),
  server = function(input, output, session) {
    
    volumes <- c(Desktop = normalizePath("~/Desktop"), Project = getwd())
    shinyFileSave(input, "saveFileUI", roots = volumes, session = session)
    shinyFileChoose(
      input = input, id = "loadFileUI", updateFreq = 0, session = session, 
      roots = volumes, filetypes = function() input$filetype, 
      pattern = function() input$filepattern
    )
    
    output$shinyFilesOutput <- renderUI({
      tagList(
        shinySaveButton(
          "saveFileUI", "Save file (renderUI)", "Save file as ...", 
          filename = input$filename, filetype = input$filetype, icon = icon("download")
        ),
        shinyFilesButton(
          "loadFileUI", "Load a file (renderUI)", "Load a file", 
          multiple = TRUE, icon = icon("upload")
        )
      )
    })
  }
)
```

**Comment by thomasp85** [comment]
Can I get you to redo this without changing quotation marks to "?

Also, I'll appreciate if this PR only contains style changes and not any change in functionality as it is impossible to review when every line is basically changed

**Comment by vnijs** [comment]
@thomasp85 and @thercast: I have now updated radiant and all radiant.* packages to use `shinyFiiles`. Radiant uses `renderUI` extensively and `shinyFiles` is working *great*! The only thing that trips me up is that double click doesn't select a file (#79) :) This is different from chromes file browser and macOS file browsers as well. @thomasp85 If you have any suggestions on how to make happen I can work on it tomorrow (Tuesday PT). If you want to try out the latest version of radiant (not from CRAN but from a miniCRAN I maintain on Github), links to install binary packages are provided below.

@thercast If I can help out with #57, please let me know. 

To install radiant.data:

```r
install.packages("radiant.data", repos = "https://radiant-rstats.github.io/minicran/")
```
Start the app using `radiant.data::radiant.data()

Or for the full Radiant package:

```r
install.packages("radiant.data", repos = "https://radiant-rstats.github.io/minicran/")
```

Start from the Rstudio Addins menu or use `radiant::radiant()`

**Comment by vnijs** [comment]
I didn't (deliberately) make changes to ' or ". Just using the `styler` default settings which is the style used in http://r4ds.had.co.nz/ and "blessed" by Hadley. 

Sorry about "stacking" the fix into the PR. The fix for #59 only affects `fileGetter` in `filechoose.R`, `DESCRIPTION` (import from methods), and `NAMESPACE` so should still be pretty easy to see I hope

File: R/filechoose.R [hunk 1]
```diff
@@ -2,376 +2,395 @@
...
     if (!is.null(filetypes)) {
-      matchedFiles <- tolower(fileInfo$extension) %in% tolower(filetypes) & fileInfo$extension != ''
+      matchedFiles <- tolower(fileInfo$extension) %in% tolower(filetypes) & fileInfo$extension != ""
       fileInfo$isdir[matchedFiles] <- FALSE
-      fileInfo <- fileInfo[matchedFiles | fileInfo$isdir,]
+      fileInfo <- fileInfo[matchedFiles | fileInfo$isdir, ]
+    }
+    if (nchar(pattern) > 0) {
+      matchedFiles <- try(grepl(pattern, fileInfo$filename), silent = TRUE)
+      if (!is(matchedFiles, "try-error")) {
```
Comment (thomasp85) [inline R/filechoose.R]: methods is generally a horrible thing to depend on (without reason) - here you can simply use `!inherits(matchedFiles, 'try-error')`

**Comment by thomasp85** [comment]
I disagree vehemently with Hadley on the use double quotation marks :-) 

It doesn't seem to be straightforward to change this in styler, so just let it be for now - I'm not going to write a lot of code in this package anymore anyway...

But please fix the methods import

**Comment by vnijs** [comment]
Done. Sorry about the " :) Following *a* coding standard is a good thing I think. Even if a few things might be a bit uncomfortable. What is the issue with `methods`?

**Comment by thomasp85** [comment]
Oh I do follow a coding standard - it just happens to be a bit different from hadley's :-)

loading methods sets forth a whole range of actions in order to build the S4 class dispatching system. Because of this it is not loaded by default when using RScript... And I have just over the years run into a lot of bugs that ultimately ended out being some sort of methods problem/incompatibility. (In this case, it is also really not needed as base::inherits provides the same functionality)

**Comment by lorenzwalthert** [comment]
I just stumbled over this. Although the PR got already merged, you can easily get rid of the rule that changes single quotes to double quotes. Here is how (for the future).
``` r
library("styler")
transformers <- tidyverse_style()
```

First, figure out where the corresponding rule lives
```r
purrr::modify_depth(transformers, 1, names)
#> $initialize
#> [1] "initialize"
#> 
#> $line_break
#> [1] "remove_line_break_before_curly_opening"            
#> [2] "remove_line_break_before_round_closing_after_curly"
#> [3] "remove_line_break_before_round_closing_fun_dec"    
#> [4] "style_line_break_around_curly"                     
#> [5] "set_line_break_after_opening_if_call_is_multi_line"
#> [6] "set_line_break_before_closing_call"                
#> [7] "remove_line_break_in_empty_fun_call"               
#> [8] "add_line_break_after_pipe"                         
#> 
#> $space
#>  [1] "indent_braces"                     
#>  [2] "unindent_fun_dec"                  
#>  [3] "indent_op"                         
#>  [4] "indent_eq_sub"                     
#>  [5] "indent_without_paren"              
#>  [6] "fix_quotes"                        
#>  [7] "remove_space_before_closing_paren" 
#>  [8] "remove_space_before_opening_paren" 
#>  [9] "add_space_after_for_if_while"      
#> [10] "add_space_before_brace"            
#> [11] "remove_space_before_comma"         
#> [12] "style_space_around_math_token"     
#> [13] "style_space_around_tilde"          
#> [14] "spacing_around_op"                 
#> [15] "spacing_around_comma"              
#> [16] "remove_space_after_opening_paren"  
#> [17] "remove_space_after_excl"           
#> [18] "set_space_after_bang_bang"         
#> [19] "remove_space_before_dollar"        
#> [20] "remove_space_after_fun_dec"        
#> [21] "remove_space_around_colons"        
#> [22] "start_comments_with_space"         
#> [23] "remove_space_after_unary_pm_nested"
#> [24] "spacing_before_comments"           
#> [25] "set_space_between_levels"          
#> [26] "set_space_between_eq_sub_and_comma"
#> 
#> $token
#> [1] "force_assignment_op"                   
#> [2] "resolve_semicolon"                     
#> [3] "add_brackets_in_pipe"                  
#> [4] "remove_terminal_token_before_and_after"
#> [5] "wrap_if_else_multi_line_in_curly"      
#> 
#> $indention
#> [1] "update_indention_ref_fun_dec"
#> 
#> $use_raw_indention
#> NULL
#> 
#> $reindention
#> [1] "indention"     "comments_only"
```
Ups, the rule lives belongs to the set of space manipulators, although it's a token modifier. Will fix that in next release.
```r
transformers$space$fix_quotes <- NULL
style_text("t <- 'this is a string'")
#> t <- "this is a string"
style_text("t <- 'this is a string'", transformers = transformers)
#> t <- 'this is a string'
```

Created on 2018-07-05 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by vnijs** [comment]
Thanks for the suggestion @lorenzwalthert. Wish I'd seen that option before

**Pull request merged**